### PR TITLE
Added SealFody sample

### PR DESCRIPTION
**Pull request by kamil-mrzyglod**

### CODE REVIEW

File: FodyAddinSamples.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.24720.0
+VisualStudioVersion = 14.0.23107.0
```
Comment (SimonCropp) [inline FodyAddinSamples.sln]: I think u need to update visual studio

File: SealSample/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
```
Comment (SimonCropp) [inline SealSample/Properties/AssemblyInfo.cs]: No need for assembly info. Can u delete

File: SealSample/SealSample.csproj [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Import Project="..\packages\Fody.1.29.4\build\dotnet\Fody.targets" Condition="Exists('..\packages\Fody.1.29.4\build\dotnet\Fody.targets')" />
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
+    <PropertyGroup>
+      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
+    </PropertyGroup>
+    <Error Condition="!Exists('..\packages\Fody.1.29.4\build\dotnet\Fody.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Fody.1.29.4\build\dotnet\Fody.targets'))" />
+  </Target>
+  <!-- To modify your build process, add your task inside one of the targets below and uncomment it.
```
Comment (SimonCropp) [inline SealSample/SealSample.csproj]: Can the redundant comment be removed

File: SealSample/SealSample.cs [hunk 4]
```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using Seal.Fody;
+
+namespace SealSample
```
Comment (SimonCropp) [inline SealSample/SealSample.cs]: I think the namespace is redundant. Can u remove

**Comment by kamil-mrzyglod** [comment]
@SimonCropp Pushed changes.

**Comment by SimonCropp** [comment]
thanks

**Pull request merged**